.github/workflows/publish-release.yml

```diff
@@ -233,7 +233,7 @@ jobs:
       run: |
         echo "## vNext (TBD)
 
-        **This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
+        **This project is in Release Candidate stage.**
```

Comment:
```suggestion
        **This project is in the Release Candidate stage.**
```
perhaps?

Reply:
I copied the text from what we already released https://pub.dev/packages/realm. I think both are acceptable.

Reply:
Not a native English speaket, but the current text seems ok to me